_data/simple-icons.json

```diff
@@ -5,11 +5,14 @@
             "hex": "ECD53F",
             "source": "https://github.com/motdotla/dotenv/blob/40e75440337d1de2345dc8326d6108331f583fd8/dotenv.svg",
             "aliases": {
-                "aka": [
-                    "Dotenv"
-                ]
+                "aka": ["Dotenv"]
             }
         },
+        {
+            "title": "Mintlify",
+            "hex": "18e299",
+            "source": "https://github.com/simple-icons/simple-icons/blob/develop/icons/mintlify.svg"
```

Comment:
You can't link to our own repository for the source 😅

Reply:
I concur - you can use the homepage of the site as your source, as there's an SVG version of the icon there.